config/routes.rb

```diff
@@ -14,6 +14,12 @@
   resources :profile_reviews, only: [:show]
   resources :successes, only: [:index]
 
+  resources :countries, only: [] do
```

Comment:
Current route:
```
states_countries GET  /countries/states(.:format)  countries#states
```
I don't know, can this be improved in `routes.rb` file. The [] is kind of weird.

Reply:
Yeh it looks strange, but its normal :)!

I would do
```
resources :countries, only: [] do
   resources :states, only: [:index] 
end
```
So you can get the countries/:country_name/states route which is the most RESTful.

Reply:
ohh I just thought of why this may extend well. Even though its currently `[]` for countries, imagine we wanted to introduce more countries? You could use countries#index to see what countries are available. Even more wild you could maybe toss a query parameter in there to filter it down by geolocation as well.

All of this is like scope creep and not worth doing now imo, but just sharing this to show where I think this could go in the future.

Reply:
Please see https://github.com/rubyforgood/pet-rescue/pull/136/commits/8ff991aff6869b34dac438f979ea10cf28fd7c5a for updated route - some things changed as well.

New route:
```
country_states GET  /countries/:country_id/states(.:format) states#index
```